browser/src/client_win.cc

```diff
@@ -14,6 +16,8 @@
 namespace content_analysis {
 namespace sdk {
 
+using NtCreateFileFn = decltype(&::NtCreateFile);
```

Comment:
Since this is only used in one function, can you move this alias to that function to improve readability?

Reply:
It's now used in two functions and there is a second function too.  So maybe I'll leave as is?